src/scss/base.scss

```diff
@@ -0,0 +1,286 @@
+            background-color: #1F242E;
+        }
+        @media (prefers-color-scheme: light) {
+            background-color: #FFFFFF;
+        }
+        @include themify($themes) {
+            background-color: themed('layoutFrontendColor');
+            color: themed('headingColor');
+        }
+    }
+
+    @include themify($themes) {
+        background-color: themed('backgroundColor');
+        color: themed('textColor');
+    }
+
+    &.full-width:not(.layout_frontend) {
+        .container {
+            min-width: 980px;
+            width: 90%;
+        }
+    }
+}
+
+.container {
+    margin: 0 auto;
+    max-width: none !important;
+    padding: 0;
+    width: 980px;
+}
+
+.page-header, .secondary-header {
+    margin-bottom: 0.5rem;
+    padding-bottom: 0.6rem;
+    @include themify($themes) {
+        border-bottom: 1px solid themed('separator');
+    }
+
+    &:not(.text-danger) {
+        h1, h2, h3, h4 {
+            margin: 0;
+            @include themify($themes) {
+                color: themed('headingColor');
+            }
+        }
+    }
+}
+
+.secondary-header, .spaced-header {
+    margin-top: 4rem;
+}
+
+.org-nav {
+    height: 100px;
+    min-height: 100px;
+    @include themify($themes) {
+        background-color: themed('navOrgBackgroundColor');
+        border-bottom: 1px solid themed('borderColor');
+        color: themed('textColor');
+    }
+
+    .container {
+        height: 100%;
+    }
+
+    .nav-tabs {
+        border-bottom: none;
+
+        a {
+            &:not(.active) {
+                border-color: transparent;
+                @include themify($themes) {
+                    color: themed('textColor');
+                }
+            }
+
+            &.active {
+                font-weight: bold;
+                padding-top: calc(#{$nav-link-padding-y} - 2px);
+                @include themify($themes) {
+                    border-top: 3px solid themed('primary');
+                    color: themed('linkColor');
+                }
+            }
+
+            &.disabled {
+                @include themify($themes) {
+                    color: themed('inputDisabledColor');
+                }
+            }
+        }
+    }
+
+    .org-name {
+        line-height: 1;
+        span {
+            display: block;
+            font-size: $font-size-lg;
+            @include themify($themes) {
+                color: themed('headingColor');
+            }
+        }
+    }
+}
+
+.page-content {
+    margin-top: 20px;
+}
+
+.footer {
+    margin-top: 40px;
+    padding: 40px 0 40px 0;
+    @include themify($themes) {
+        border-top: 1px solid themed('separator');
+    }
+}
+
+hr, .dropdown-divider {
+    @include themify($themes) {
+        border-top: 1px solid themed('separatorHr');
+    }
+}
+
+.min-height-fix {
+    min-height: 1px;
+}
+
+.overflow-hidden {
+    overflow: hidden;
+}
+
+.cursor-move {
+    cursor: move !important;
+}
+
+.bg-primary {
```

Comment:
Are all these uses of `!important` necessary? You should generally use specificity to resolve competing CSS rules, `!important` is considered bad practice and can make stylesheets difficult to maintain. Please try to refactor this document so that you're not using `!important` (unless it was already there before your changes)